docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+1. [Technical Requirements and the Installation of the Docs](/3.0.0-beta.x/community/contribute-with-docs.html#technical-requirements)
+2. [Understanding the Docs Structure](/3.0.0-beta.x/community/contribute-with-docs.html#understanding-the-docs-structure)
+3. [Submitting new documentation](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-totally-new-documentation)
+4. [Adding/modifying/deleting documentation](/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content)
+
+- How to [submit an externally hosted tutorial](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-an-externally-hosted-tutorial).
+- How to use the [Docs Style Guide](/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) to write better Docs.
+
+### Technical Requirements
+
+1. You have [Node v10.x.x](https://nodejs.org/en/download/) (LTS) or [Node v12.x.x](https://nodejs.org/en/download/current/) and [Yarn v1.2.0+](https://yarnpkg.com/en/).
+2. You are familiar with Git, and have it installed on your development environment.
+
+- Ensure you're using the [required versions of Node.js and npm](/3.0.0-beta.x/getting-started/install-requirements.html).
+- Fork the [Strapi repository](https://github.com/strapi/strapi) [to your own GitHub account](https://help.github.com/en/articles/fork-a-repo).
+- Clone your Strapi repository:
+
+`Path: ./Projects/`
+
+```bash
+git clone git@github.com:YOUR_USERNAME/strapi.git
+```
+
+- Install the dependencies **for the docs** section of the monorepo:
+
+`Path: ./Projects/`
+
+```bash
+cd strapi/docs
+npm install
+```
+
+- Start the docs development server and open the documentation in the browser:
+
+```bash
+npm run dev
+```
+
+- Then, navigate to [http://localhost:8080/documentation/](http://localhost:8080/documentation/).
+
+You should now have a running instance of the documentation. The next section explains the organization and structure of the documentation.
+
+### Understanding the Docs Structure
+
+The Strapi docs are power by [VuePress](https://vuepress.vuejs.org/). The docs live in a folder called **docs**. `Path: ./strapi/docs`. The _VuePress_ installation lives in this folder. It has a separate installation and separate dependencies from the rest of the Strapi monorepo.
+
+Only version **3.0.0-beta.x** is currently maintained. So, therefore, only the documentation for this version is maintained. The documentation is located at `./strapi/docs/3.0.0-beta.x/`. Here you find all the content and assets for the documentation.
+
+The _folder names_ in the file directory correspond to the different sections in the left-hand menu. For example, the folder called, "Getting Started," corresponds to the section called, "Getting Started." However, the `h1` tag at the top of the files control the left-menu titles.
+
+For example, if you add a page or folder, you do it in the config file of the Beta docs. `Path: ./strapi/docs/.versions/3.0.0-beta.x.js/`
```

Comment:
The path doesn't exist

Reply:
I clarified the links in this section and cleaned up the example.